docs/api/process.md

```diff
@@ -68,6 +77,11 @@ instead of the `--no-deprecation` command line flag.
 
 A `String` representing the path to the resources directory.
 
+### `process.sandboxed`
+
+A `Boolean`. When the renderer process is sandboxed, this property is `true`,
+otherwise it is `undefined`.
```

Comment:
How difficult is it to make it to be equal to `false` in a non-sandboxes renderer?

Reply:
@alexeykuzmin it's simple of course. I did it like that as some other properties follow similar pattern (only being set to true if applicable, missing otherwise)